fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/export/fast/NopWriter.java

```diff
@@ -27,7 +27,7 @@
     @Inject
     JobContext jobContext;
 
-    /**
+    /*
```

Comment:
why not javadoc?

Reply:
it's a private method

Reply:
the next  method was private

Reply:
private methods can have javadoc.  it makes it come up nice from your IDE.  it doesn't go into the published javadoc.  I'd vote to keep using javadoc-style comments for private methods.